<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGroupDialog</class>
 <widget class="QDialog" name="AddGroupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить группу</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLineEdit" name="groupNumberEdit">
     <property name="placeholderText">
      <string>Номер группы</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="studentsEdit">
     <property name="placeholderText">
      <string>Имена студентов (каждое с новой строки)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Курс:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="courseNumber"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Лекции:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="lecturesSpinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Практики:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="practicesSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>AddGroupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AddGroupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
